packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -325,6 +326,9 @@ class GestureDetector extends StatelessWidget {
   /// duplication of information.
   final bool excludeFromSemantics;
 
+  /// {@macro flutter.gestures.recognizer.dragStartBehavior}
+  final DragStartBehavior startBehavior;
```

Comment:
this one in particular should almost certainly be dragStartBehavior because otherwise it'll be confusing to have a `behavior` and a `startBehavior` that mean different things.

Reply:
So for some reason, values weren't passed correctly when I used dragStartBehavior. Something about doing the ..dragStartBehavior = dragStartBehavior didn't work the way I wanted it to... I'll try it again though.